package.json

```diff
@@ -14,14 +14,19 @@
     "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.9.28"
+    "mongoose": "^5.9.28",
+    "passport": "^0.4.1",
+    "passport-google-oauth": "^2.0.0"
   },
   "devDependencies": {
     "@commitlint/cli": "^9.1.2",
     "@types/cors": "^2.8.7",
     "@types/jest": "^26.0.13",
     "@types/mongoose": "^5.7.36",
     "@types/node": "^14.6.4",
+    "@types/passport": "^1.0.4",
+    "@types/passport-google-oauth": "^1.0.41",
+    "@types/passport-google-oauth2": "^0.1.3",
```

Comment:
What is the difference and do we need both?

Reply:
the diffrerence berween both libraries is   
oAuth 1.0 accept  consumer key, consumer secret, and callback URL
OAuth 2.0 accept  client ID, client secret, and callback URL, i will delete @types/passport-google-oauth2.